perfdash/config.go

```diff
@@ -282,24 +282,14 @@ var (
 		"APIServer": {
 			"Responsiveness": []TestDescription{
 				{
-					Name:             "pod-with-ephemeral-volume-startup-latency",
-					OutputFilePrefix: "APIResponsiveness",
-					Parser:           parsePerfData,
-				},
-				{
-					Name:             "storage-",
+					Name:             "storage",
```

Comment:
Do the "old storage" test still work after this change?

Reply:
It seems to work properly:
* I've started the perfdash locally and it has both the "old storage" metrics and "storage suite" metrics
* Checked files from old test run (https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-kubernetes-storage-scalability-max-downwardapi-vol-per-node/1178934805790724098/artifacts/) and each it looks like using "storage-" or "storage" prefixes should have same effect